backend/RepositoryLayer/ModificationEntryRepository.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ModelLayer;
+using ModelLayer.Helper;
+using ModelLayer.Models;
+using RepositoryLayer.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RepositoryLayer
+{
+    public interface IModificationEntryRepository : IBaseRepository<ModificationEntry>
+    {
+        Task UpdateModificationAsync(long id, MODEL_TYPE dataType);
+        Task CreateNewEntryAsync(long id, MODEL_TYPE dataType);
+        Task RemoveEntryAsync(long id, MODEL_TYPE dataType);
+        Task<List<ModificationEntry>> GetSortedModificationEntriesByTypeAsync(MODEL_TYPE dataType);
```

Comment:
- Wieso gibt es hierfür neue Funktionen und werden nicht die vom BaseRepository benutzt?
- Warum haben die Funktionen keinen Rückgabewert wie zum Beispiel IdentityResult?
- Wieso werden die IDs selber gesetzt?

Reply:
bei der id handelt es sich nicht um die id des zu erzeugenden objekts in der db sondern um die referenzierte model id die verändert wurde

Reply:
und naja, klar rufe ich deine repo funktionen auf, aber davor muss ich halt noch weitere sachen erledigen, wofür ich schon der meinung bin das es legitim ist funktionen zu erstellen